Chinwe Footer/FAQ

bianca-sparxs opened the pull request:
# Description

The Footer/FAQ sections updated in line with website V3.5; the new sponsor section has Google, Twillio, Liberty, and Giphy logos; FAQ section is in aligned in a 3x3 grid, Flair elements added. Twitter and Facebook links should take your to respective page's social media. Doing 'Back to Top' link by tomorrow.

# Reviewing
See if flair elements don't mess with the grid

# Tests
I affirm that I've tested my code on:
- [x] Desktop
- [x] Tablet
- [x] Mobile
(Chrome tools counts)

Commit:
Added Hero Text Component
HeroText in components folder, has one blue wave

Commit:
Added Waves

Commit:
Merge branch 'master' of github.com:Bostonhacks/beta into chinweHeroText

Commit:
Set up images to have grid in them

Commit:
Merge branch 'master' of github.com:Bostonhacks/beta into chinweHeroText

Commit:
Made Footer Wave Curves one SVG
started the grid to put design elements, sponsorship logos

Commit:
Added Sponsor Logos, Design Underway
Sponsorship logos added in assets, add design elements on FAQ and Grid Sections

Commit:
Added LogoFooter Compnonent
Contains back to top button and social media links

Commit:
Merge branch 'master' of github.com:Bostonhacks/beta into chinweHeroText

Commit:
Links Added, Flair Added

maridematte reviewed (changes requested):
There is an alignment issue on bigger screens between the back to top button and the social media buttons. The flair doesn't quite adjust to the screen in smaller sizes. And at this stage we need to worry about the mobile version, which is good for the FAQ but needs a bit more work on the sponsor logos on the footer.

Commit:
Fixed Scaling, Back to Top Link Added
sponorship logos and text should not scale down/up so much for website as well as spacing.

maridematte commented:
Looks good! The only thing now is that the back to top button doesn't quite scale into mobile and it looks extremely small

Commit:
Back to Top Fix,  Mid Curve is one SVG

Commit:
Back to Top Fix

Commit:
Merge branch 'master' of github.com:Bostonhacks/beta into chinweHeroText
